explorer/templates.go

```diff
@@ -315,7 +315,7 @@ func makeTemplateFuncMap(params *chaincfg.Params) template.FuncMap {
 		"toLowerCase": func(a string) string {
 			return strings.ToLower(a)
 		},
-		"fetchRowLinkURL": func(groupingStr string, start, end types.TimeDef) string {
+		"fetchRowLinkURL": func(groupingStr string, start, end time.Time) string {
```

Comment:
Note that this is a current bug on master that is preventing any of the time-based block pages (/weeks, /months, etc.) from loading.